### ISSUE TITLE
Alternate Weapons incorrect with PlayerHurt event

### ISSUE THREAD

**Issue by allenymin** [markus-wa/demoinfocs-golang#148]
There seems to be a bug with the weapon id when using alternate weapons in the player hurt event. 
When using an alternate weapon like the USP-S, WeaponFire and Kill events will properly show USP-S as the weapon, but the PlayerHurt event will give P2000 as the weapon.

Any idea why this might be happening?

**Comment by BestAwperEver** [comment]
Btw [this](https://github.com/markus-wa/demoinfocs-golang/issues/148#issuecomment-530944671) seems to be working:
```
DEBUG: tick 112596: keev bought p250 (P250) AM:13 AR:26 AM2:13 AR2:26
DEBUG: tick 116886: autimatic bought p250 (CZ75 Auto) AM:0 AR:0 AM2:0 AR2:0
DEBUG: tick 117613: n0thing bought p250 (P250) AM:13 AR:26 AM2:13 AR2:26
DEBUG: tick 118894: Skadoodle bought p250 (CZ75 Auto) AM:0 AR:0 AM2:0 AR2:0
```

**Comment by markus-wa** [comment]
Should be fixed with [v1.3.2](https://github.com/markus-wa/demoinfocs-golang/releases/tag/v1.3.2) 🙂
